tests/integration/sentinel/transitions/sentinel-transition-error-log.spec.js

```diff
@@ -0,0 +1,27 @@
+const utils = require('@utils');
+const sentinelUtils = require('@utils/sentinel');
+const { setTimeout } = require('timers/promises');
+const { expect } = require('chai');
+
+describe('Sentinel transition error log', async function() {
+  this.timeout(10 * 60 * 1000);
+  after(async () => await utils.revertSettings(true));
+
+  it('log error for unkown transition', async () => {
+    const unknownTransitionPattern = /Unknown transition "something"/;
+    const settings = {
+      transitions: {
+        something: true
+      }
+    };
+
+    const collectLogs = await utils.collectSentinelLogs(unknownTransitionPattern);
+    await utils.updateSettings(settings, 'sentinel');
+    await sentinelUtils.getCurrentSeq();
+    const logs = await collectLogs();
+    expect(logs).to.have.lengthOf(1);
+    await setTimeout(5 * 60 * 1000); // logs every 5 minutes
```

Comment:
Oh... :( Adding 5 minutes of waiting to test time is _really_ expensive. Is there any way we can get this without waiting for 5 minutes?

Reply:
since there is a unit test that covers that, I am removing it here to save on build time